src/inc/corjit.h

```diff
@@ -121,34 +120,10 @@ enum CorJitFlag
 
 #endif // !defined(_TARGET_X86_) && !defined(_TARGET_AMD64_)
 
-#if defined(MDIL)
-
-    CORJIT_FLG_MDIL                = 0x00004000, // Generate MDIL code instead of machine code
-
-    // Safe to overlap with CORJIT_FLG_MAKEFINALCODE below. Not used by the JIT, used internally by NGen only.
-    CORJIT_FLG_MINIMAL_MDIL        = 0x00008000, // Generate MDIL code suitable for use to bind other assemblies.
-
-    // Safe to overlap with CORJIT_FLG_READYTORUN below. Not used by the JIT, used internally by NGen only.
-    CORJIT_FLG_NO_MDIL             = 0x00010000, // Generate an MDIL section but no code or CTL. Not used by the JIT, used internally by NGen only.
-
-#else // defined(MDIL)
-
     CORJIT_FLG_CFI_UNWIND          = 0x00004000, // Emit CFI unwind info
-
-#if defined(FEATURE_INTERPRETER)
```

Comment:
You dropped this `FEATURE_INTERPRETER` ifdef... just checking this was intentional.